sorc/build_ufs.sh

```diff
@@ -13,7 +13,7 @@ source "${cwd}/ufs_model.fd/tests/module-setup.sh"
 
 while getopts ":da:v" option; do
   case "${option}" in
-    d) BUILD_TYPE="Debug";;
+    d) BUILD_TYPE="DEBUG";;
```

Comment:
I think this should be kept `Debug` (as that is the recognized `CMAKE_BUILD_TYPE` and an edit be made on line 31 as:
```
[[ ${BUILD_TYPE:-"Release"} = "Debug" ]] && MAKE_OPT+=" -DDEBUG=ON"
```

Reply:
> I think this should be kept `Debug` (as that is the recognized `CMAKE_BUILD_TYPE` and an edit be made on line 31 as:
> 
> ```
> [[ ${BUILD_TYPE:-"Release"} = "Debug" ]] && MAKE_OPT+=" -DDEBUG=ON"
> ```

Whichever way is better works for me.  I had just noticed that in the other "sorc/build*.sh" scripts that the equivalent to line 16 always has "DEBUG" in all caps, i.e.,  "d) export BUILD_TYPE="DEBUG";;"

I'll stand by until I hear from the other reviewers about which way to go.

Reply:
Thanks for checking.
In the interest of getting this in, we can do a consistency review at a later time.

Reply:
It seems that this case only effects the internals of the "build_ufs.sh" script, and wouldn't have an effect anywhere else.  But that's based on my limited knowledge of the workflow as a whole.